tests/cases/fourslash/codeFixPromiseToAsyncMultipleThensSameVarName.ts

```diff
@@ -0,0 +1,39 @@
+/// <reference path='fourslash.ts' />
+
+// @target: es6
+
+////function [|f|]():Promise<boolean> {
+////    return fetch('http://yahoo.com').then(res).then(res2);
+////}
+////
+////function res(result){
+////    return result.ok;
+////}
+////
+////function res2(result){
+////    console.log(result);
+///}
+
+verify.getSuggestionDiagnostics([{
+    message: "This may be converted to use async and await.",
+    code: 80006,
+}]);
+
+verify.codeFix({
+    description: "Convert to use async and await",
+    index: 0,
+    newFileContent:
+`async function f():Promise<boolean> {
+    let result = await fetch('http://yahoo.com');
+    let temp = await res(result);

```

Comment:
Hmm, ginning up temporaries seems hard. Are these like template holes so I can tab to them and give them a proper name easily?